HyperV/scripts/export-eventlog.ps1

```diff
@@ -1,27 +1,47 @@
-function dumpeventlog(){
+# Loading config
 
-	get-eventlog -list | ForEach-Object {
+. "C:\OpenStack\nova-ci\HyperV\scripts\config.ps1"
+
+function dumpeventlog($path){
+	
+	Get-Eventlog -list | ForEach-Object {
 		$logFileName = $_.LogDisplayName
-		$path = "C:\OpenStack\Logs\Eventlog\"
-		$exportFileName = "eventlog_" + $logFileName + (get-date -f yyyyMMdd) + ".evt"
+		$exportFileName =$path + "\eventlog_" + $logFileName + (get-date -f yyyyMMdd) + ".evt"
 		$exportFileName = $exportFileName.replace(" ","_")
 		$logFile = Get-WmiObject Win32_NTEventlogFile | Where-Object {$_.logfilename -eq $logFileName}
-		$logFile.backupeventlog($path + $exportFileName) -ErrorAction SilentlyContinue
-		Clear-Eventlog "$logFileName"
+		try{
+			$logFile.backupeventlog($exportFileName)
+		} catch {
+			break
```

Comment:
you are aware that the "break" will exit the foreach loop and any remaining eventlogs will not be saved?